### ISSUE TITLE
Fileserver fails to start

### ISSUE THREAD

**Issue by morris-jason** [vmware/vic-product#1505]
##### For bug reports, please include the information below:

REQUIRED:

- VC 6.7, Embedded PSC
- Dev build of v1.4.0-dev-0-1a06de7d

OPTIONAL, but helpful:
- 10.161.175.100

Bug Report Detailed Information
-------------------------------

- What operation was being performed when the failure was noticed?

Fileserver fails to start with the following error after system reboot.

```
Mar 22 19:03:32 Photon bash[4594]: gzip: stdin: unexpected end of file
Mar 22 19:03:32 Photon bash[4594]: tar: Unexpected EOF in archive
Mar 22 19:03:32 Photon bash[4594]: tar: Unexpected EOF in archive
Mar 22 19:03:32 Photon bash[4594]: tar: Error is not recoverable: exiting now
```

- Provide additional detail as necessary:

I have experienced this during upgrade as well. This time it took by 15 reboots to reproduce, but in the past it has happened on first boot.

#### Acceptance criteria

- [x] Fileserver should boot up to provide registration services even if the `vic_*.tar.gz` archive in `/opt/vmware/fileserver/files` fails to extract.
- [ ] Investigate why reboots are corrupting the archive.
- [ ] The error may also be caused by a script messing with `/tmp/ `while the fileserver is writing the `vic_*.tar.gz` archive in `configure_fileserver.sh.`

**Comment by morris-jason** [comment]
Broken tar extraction
```
root@Photon [ /opt/vmware/fileserver/files ]# tar -xvzf vic_17635.tar.gz
vic/
vic/README
vic/vic-machine-darwin
vic/appliance.iso
vic/vic-ui-windows.exe
vic/vic-machine-linux
vic/vic-machine-windows.exe
vic/vic-ui-darwin
vic/vic-ui-linux
vic/bootstrap.iso

gzip: stdin: unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
```

Successful tar extraction on a master build:
```
root@Photon [ /opt/vmware/fileserver/files ]# tar -zxvf vic_17719.tar.gz
vic/
vic/README
vic/vic-machine-darwin
vic/appliance.iso
vic/vic-ui-windows.exe
vic/vic-machine-linux
vic/vic-machine-windows.exe
vic/vic-ui-darwin
vic/vic-ui-linux
vic/bootstrap.iso
vic/LICENSE
vic/vic-machine-server
vic/ui/
vic/ui/plugin-packages/
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugins/
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugins/gson-2.3.1.jar
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugins/vic-service.jar
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugins/vim25.jar
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugins/vic.war
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051/plugin-package.xml
vic/ui/plugin-packages/com.vmware.vic-v1.3.1.1051.zip
vic/ui/sync-vic-ui-version.sh
vic/ui/plugin-manifest
vic/ui/VCSA/
vic/ui/VCSA/configs
vic/ui/VCSA/install.sh
vic/ui/VCSA/upgrade.sh
vic/ui/VCSA/uninstall.sh
vic/ui/vsphere-client-serenity/
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051.zip
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/plugins/
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/plugins/vic-ui-service.jar
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/plugins/vic-ui-war.war
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/plugins/vim25.jar
vic/ui/vsphere-client-serenity/com.vmware.vic.ui-v1.3.1.1051/plugin-package.xml
vic/ui/vCenterForWindows/
vic/ui/vCenterForWindows/configs
vic/ui/vCenterForWindows/uninstall.bat
vic/ui/vCenterForWindows/utils/
vic/ui/vCenterForWindows/utils/xml.exe
vic/ui/vCenterForWindows/install.bat
vic/ui/vCenterForWindows/upgrade.bat
```

**Comment by morris-jason** [comment]
Removing p0, work around is to stop the fileserver service and download a fresh vic tar.

**Comment by morris-jason** [comment]
This issue has been partially fixed in #1529. Investigation is still needed as to how the archive gets corrupted in the first place. 

The customer no longer needs to download a new tar if they experience this issue, as the original tar is never altered. The new workaround is a fileserver restart, or a guest os reboot.
